src/v/ssx/future-util.h

```diff
@@ -13,12 +13,17 @@
 #include "utils/concepts-enabled.h"
 #include "utils/functional.h"
 
+#include <seastar/core/abort_source.hh>
+#include <seastar/core/condition-variable.hh>
 #include <seastar/core/future-util.hh>
 #include <seastar/core/future.hh>
+#include <seastar/core/gate.hh>
+#include <seastar/core/semaphore.hh>
 #include <seastar/core/when_all.hh>
 
 #include <algorithm>
 #include <iterator>
+#include <tuple>
```

Comment:
Is this needed?

Reply:
It is not, that was a hangover from when I tried out std::ignore stuff, will remove.